compiler/executable/pipeline.rs

```diff
@@ -202,52 +234,43 @@ pub(crate) fn compile_pipeline_stages(
     annotated_stages: &[AnnotatedStage],
     input_variables: impl Iterator<Item = Variable>,
     function_return: Option<&[Variable]>,
-) -> Result<(HashMap<Variable, VariablePosition>, Vec<ExecutableStage>, Cost), ExecutableCompilationError> {
+) -> Result<(HashMap<Variable, VariablePosition>, Vec<ExecutableStage>, TypePopulations), ExecutableCompilationError> {
```

Comment:
The `Cost` was meant to be used for plan cache eviction. Its role is filled by `TypePopulations`.